app/src/main/java/edu/uc/coffeens/stocktracker/services/StockService.kt

```diff
@@ -15,12 +16,12 @@ class StockService {
      */
     fun fetchStocks(): MutableLiveData<ArrayList<Stock>> {
 
-        var _stocks = MutableLiveData<ArrayList<Stock>>()
+        val _stocks = MutableLiveData<ArrayList<Stock>>()
```

Comment:
_stocks is only ever declared and used once, so it can be a val rather than a var.